src/tests/test_typeconversions.py

```diff
@@ -799,14 +799,15 @@ def test_pythonexc2js(selenium):
 @run_in_pyodide
 def test_js2python_null(selenium):
     from pyodide.code import run_js
-
-    assert run_js("null") is None
-    assert run_js("[null]")[0] is None
-    assert run_js("() => null")() is None
-    assert run_js("({a: null})").a is None
-    assert run_js("new Map([['a', null]])")["a"] is None
-    assert run_js("[null, null, null]").to_py() == [None, None, None]
-    assert run_js("new Map([['a', null]])").to_py() == {"a": None}
+    from pyodide.ffi import jsnull
+
+    assert run_js("null") is jsnull
```

Comment:
This is going to be breaking change, so we should mention it in the changelog clearly.